<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Puerto:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="puerto_spinBox">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="conectar_pushButton">
         <property name="text">
          <string>Conectar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="desconectar_pushButton">
         <property name="text">
          <string>Desconectar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="datos_listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data Length:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="size_spinBox">
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
